pkg/dnfjson/dnfjson.go

```diff
@@ -776,7 +783,7 @@ func run(dnfJsonCmd []string, req *Request) ([]byte, error) {
 	cmd := exec.Command(ex, args...)
 	stdin, err := cmd.StdinPipe()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("calling %s failed: %w", ex, err)
```

Comment:
This might be a bit misleading, the error is from `StdinPipe` so maybe instead of `calling %s failed` something like `creating stdin pipe failed` or similar ?